src/searching.md

```diff
@@ -387,27 +388,26 @@ you need to tell Anki not to treat them specially.
   use `\\` instead of `\`.
 
 - `(` and `)`\
-  You can search for parentheses either by enclosing the full term in quotes,
-  and/or by using a backslash. That is, `"some(text)"`, `some\(text\)` and
-  `"some\(text\)"` are all equivalent, but `some(text)` is not.
+  You can search for parentheses by enclosing the entire term in quotes,
+   by using a backslash, or both at the same time. For example, `"(text)"`, `\(text\)` and
+  `"\(text\)"` are all equivalent searches, and searches for `(text)`.
 
 - `-`\
   Starting a search term with `-` usually inverts it: `-dog` matches everything
   except dog for example. If you instead wish to include an actual hyphen,
-  you can either use a backslash, or include the text in quotes, such as
-  `\-.-` or `"-.-"`.
+  you can either use a backslash, or include the text in quotes. For example,
+  `\-free` or `"-free"` will match "guilt-free" and "cruelty-free".
 
 - `:`\
-  Colons have to be escaped unless they are preceded by another, unescaped colon.
-  So `w:e:b` is a word boundary search for `e:b`, `w\:e\:b` searches literally for
-  `w:e:b` and `w\:e:b` searches the field `w:e` for `b` (see
-  [field searches](#limiting-to-a-field)).
+  Colons have to be escaped using backslashes unless they are preceded by another, unescaped colon.
+  For example, `w:3:30` searches for 3:30 on word boundary and doesn't require you to use a backslash.
```

Comment:
Should use quotation marks, to match other examples:
`searches for "3:30" on word boundary`

Reply:
```suggestion
  For example, `w:3:30` searches for "3:30" on word boundary and doesn't require you to use a backslash.
```